Kernel/FileSystem/ext2_fs.h

```diff
@@ -1,733 +0,0 @@
-/*
- *  linux/include/linux/ext2_fs.h
- *
- * Copyright (C) 1992, 1993, 1994, 1995
- * Remy Card (card@masi.ibp.fr)
- * Laboratoire MASI - Institut Blaise Pascal
- * Universite Pierre et Marie Curie (Paris VI)
- *
- *  from
- *
- *  linux/include/linux/minix_fs.h
- *
- *  Copyright (C) 1991, 1992  Linus Torvalds
```

Comment:
I don't think we want to re-organize this file out into our own custom definitions file, it's GPL iirc. We should keep it as-is. Or at least, reimplement it completely while looking only at an ext2 spec document. I think that should be done in a separate PR.

Reply:
It will not pass CI otherwise... or maybe it's just that I need to update the lint scripts to not look into this updated location.